cryptoki/src/mechanism/aead.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Contributors to the Parsec project.
```

Comment:
Is this the correct year? Was the file here before?

Reply:
:thinking: nope, actually I just copied it from the other file not being sure what's the procedure :sweat_smile: 

These date ranges are always problematic and I've seen [other projects remove them altogether](https://github.com/curl/curl/pull/10205). :shrug: 

(For this PR I fixed what you reported to keep this consistent).